docs/cues/creating-a-cue.md

```diff
@@ -8,25 +8,29 @@ How Titan Works When Programming
 --------------------------------
 
 When you select one or more dimmers or fixtures for control, they are
-loaded into the ***Editor***. You can then use the 
-[wheels](../controlling-fixtures/using-the-select-buttons-and-wheels.md#changing-attributes-using-the-wheels)
-and [palettes](../palettes.md) to
-change the settings on the fixture. You can also apply shapes to it.
-
-If a fixture is selected after you have changed some attributes then the
-current list of fixtures in the Editor is emptied and a new list is
-started.
-
-All fixtures and attributes that have been edited since the last Clear
-are stored in the ***Programmer***. The order in which you selected the
-fixtures is also stored, and is used with [shapes](../effects.md) and the
+loaded into the **Programmer**. You can then use the 
+[wheels](../controlling-fixtures/using-the-select-buttons-and-wheels.md#changing-attributes-using-the-wheels),
+[palettes](../palettes.md) and [effects](../effects.md) to 
+change the attributes on the fixture and your changes will be
+stored in the Programmer.
+
+The order in which you selected the
+fixtures is also stored, and is used with [effects](../effects.md) and the
 [Fixture Overlap](cue-timing.md#setting-fade-times-and-overlap-for-a-cue)
-function. When you record a cue, the contents of the Programmer are
-saved into the cue.
+function. 
 
-When you press \<Clear\>, the programmer and editor are emptied. This
+When you record a cue, the contents of the Programmer are
+saved into the cue. This might be different to what you see on stage as you
+are only saving your changes, not the output of the console. See **Record
+Mode** in the next section for different options of what gets saved.
```

Comment:
Could link to the record mode section

Reply:
There isn't a section on it, it's just described in a bullet point below